client/blocks/product-purchase-features-list/sell-online-paypal.jsx

```diff
@@ -19,7 +19,7 @@ export default localize( ( { isJetpack, translate } ) => {
 	return (
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
-				buttonText={ translate( 'Sell online' ) }
+				buttonText={ translate( 'Collect payments' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Collect Payments and Donations' )` *ES Score: 7*